mindsdb/api/mysql/mysql_proxy/mysql_proxy.py

```diff
@@ -181,12 +233,30 @@ def get_fast_auth_password():
         if handshake_resp.type == 'SSLRequest':
             log.info('switch to SSL')
             self.session.is_ssl = True
+
+            use_temp_cert = isinstance(CERT_PATH, str) is False or len(CERT_PATH) == 0
+            cert_path = self.make_temp_cert() if use_temp_cert else CERT_PATH
```

Comment:
I don't think this is the way we want to go about thing. Why not generate a cert on install and stick with it ?

Generating a new cert for every ssl request we get seem very wasteful. Am I getting something wrong here ?

Reply:
changed to make cert on install